spring-integration-amqp/src/test/java/org/springframework/integration/amqp/config/AmqpChannelParserTests.java

```diff
@@ -57,7 +58,7 @@ public void interceptor() {
 				TestUtils.getPropertyValue(channel, "dispatcher"), "maxSubscribers", Integer.class).intValue());
 		channel = context.getBean("pubSub", MessageChannel.class);
 		Object mbf = context.getBean(IntegrationUtils.INTEGRATION_MESSAGE_BUILDER_FACTORY_BEAN_NAME);
-		assertSame(mbf, TestUtils.getPropertyValue(channel, "dispatcher.messageBuilderFactory"));
+		assertNotSame(mbf, TestUtils.getPropertyValue(channel, "dispatcher.messageBuilderFactory"));
```

Comment:
Oh, I see; you're not saying that at runtime we're using a different builder just that, since these are parser only tests, we won' have it yet.

I agree; they should just be removed.

However, we might want to consider adding verification to runtime tests instead :smile: 

(That can be a separate PR if you prefer)